components/builder-db/src/pool.rs

```diff
@@ -79,12 +79,7 @@ impl Pool {
         shard_id: u32,
     ) -> Result<r2d2::PooledConnection<r2d2_postgres::PostgresConnectionManager>> {
         let conn = self.inner.get().map_err(Error::ConnectionTimeout)?;
-        debug!("Switching to shard {}", shard_id);
-
-        let schema_name = format!("shard_{}", shard_id);
-        let sql_search_path = format!("SET search_path TO {}", schema_name);
-        conn.execute(&sql_search_path, &[])
-            .map_err(Error::SchemaSwitch)?;
+        debug!("Ignoring shard {} on purpose", shard_id);
```

Comment:
Cool.. it would also be nice to not see any more shard messages in debug mode.. maybe make this a trace message?

Reply:
One other comment - it might be good to remove the shards field from the Pool structure entirely.  This will help catch other changes - for example, there is cross-DB sync code (eg, sync packages) that is iterating over all shards that should be adjusted to not iterate.

Reply:
Sure

Reply:
Ah, good point.  Will do.